README.md

```diff
@@ -9,7 +9,7 @@ This add-on aims at providing a configuration layer to let implementers start an
 
 Settings and metadata will be loaded onto the Odoo server based on what is provided in the data files.
 
-Those data files are mostly in CSV and XML formats, depending of the model that is loaded.
+Those data files are mostly in CSV and XML formats, depending on the model that is loaded.
```

Comment:
I'm skeptical about this change but I guess the grammar should be `depending on` instead of `depending of`